### PR TITLE
Add Buildah builds itself blog post.

### PR DESCRIPTION
**Pull request by sanjabonic**

### CODE REVIEW

File: source/blog/2018-02-27-buildah-builds-itself.html.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+# Install build dependencies and build binary packages.
+buildah --debug=false run $cid -- dnf -y install 'dnf-command(builddep)' rpm-build
+buildah --debug=false run $cid -- dnf -y builddep --spec rpmbuild/SPECS/buildah.spec
+buildah --debug=false run $cid -- rpmbuild --define "_topdir /rpmbuild" -ba /rpmbuild/SPECS/buildah.spec
+```
+
+Then a second container is built again using Fedora 27 and the rpm file that was built on the first container is copied into this new container. Then the rpm is installed using dnf. Buildah has now been installed onto the new container with the latest bits from GitHub.
+
+```
+	# Build a second new container.
```
Comment (TomSweeneyRedHat) [inline source/blog/2018-02-27-buildah-builds-itself.html.md]: Please remove the whitespace/tab before the comment here.

File: source/blog/2018-02-27-buildah-builds-itself.html.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+echo "id: $id"
+echo "buildah version: $bv"
+echo "buildah rpm version: $rv"
+test $shortcommit = $id
+test $bv = $rv
+```
+
+There you have it, Buildah building and testing itself.  Granted I’m not too worried if Buildah doesn’t follow the "Three Laws of Robotics" at this point, but this technique is really powerful. This test runs on each and every pull request to Buildah. If the RPM can’t be built, installed and minimal operations completed, the tests won’t run and the code won’t get committed.
+
+For more information on Buildah check it out on [GitHub](https://github.com/projectatomic/buildah) - also checkout the [rpm.bats](https://github.com/projectatomic/buildah/blob/master/tests/rpm.bats) file there which is where code examples in this blog came from.  If you’re interested, we’d love to have you become a contributor on the product.
```
Comment (TomSweeneyRedHat) [inline source/blog/2018-02-27-buildah-builds-itself.html.md]: My bad.  Could you change the last word "product" to "project" please?

**Comment by rhatdan** [comment]
When is this going to be published?

**Comment by mscherer** [comment]
It is already here: http://www.projectatomic.io/blog/2018/02/buildah-builds-itself/ 

It usually build in 5 to 10 minutes after merge, unless there is a issue, and we have to restart the build, then someone with access to the openshift account should restart. Feel free to ping me if that happen.

**Pull request merged**